pmd-dist/src/test/java/net/sourceforge/pmd/it/BinaryDistributionIT.java

```diff
@@ -33,8 +33,8 @@ class BinaryDistributionIT extends AbstractBinaryDistributionTest {
         "fortran", "gherkin", "go", "groovy", "html", "java", "jsp",
         "julia",
         "kotlin", "lua", "matlab", "modelica", "objectivec", "perl",
-        "php", "plsql", "python", "ruby", "scala", "swift", "tsql",
-        "typescript", "vf", "xml"
+        "php", "plsql", "pom", "python", "ruby", "scala", "swift",
+        "tsql", "typescript", "vf", "vm", "wsdl", "xml", "xsl"
```

Comment:
Looks like we got some new languages :)

I think we should update VM to return a proper tokenizer (since it is a javacc based language). And pom/wsdl/xsl should return a XmlTokenizer (like xml).